src/styles/blocks.scss/clock.scss

```diff
@@ -0,0 +1,61 @@
+.clock {
+  display: flex;
+  justify-content: center;
+
+  &__face {
+    position: relative;
+
+    display: flex;
+    @include circle($size);
+
+    background-color: #d1d1d1;
+    border-radius: 50%;
+    border: 5px solid black;
+  }
+  &__hand-minute {
+    @include hand-position;
+
+    height: $hand-size;
+    width: 10px;
+
+    background-color: black;
+    animation: clock-rotate 3600s linear infinite;
+  }
+
+  &__hand-second {
+    @include hand-position;
+
+    height: $hand-size;
+    width: 5px;
+
+    background-color: red;
+    animation: clock-rotate 60s linear infinite;
+  }
+
+  &__center {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    transform: translate(-50%, 50%);
+
+    height: 50px;
+    width: 50px;
+
+    border-radius: 50%;
+    background-color: red;
+  }
+
+  @keyframes clock-rotate {
+    from {
+      transform: rotate(0deg);
+      transform-origin: bottom center;
+    }
+
+    to {
+      transform: rotate(360deg);
+      transform-origin: bottom center;
+    }
+  }
```

Comment:
move `transform-origin` to the class itself, no need to set it in the animation 😉